topics/introduction/tutorials/vsi_qc/tutorial.md

```diff
@@ -0,0 +1,244 @@
+{: .comment}
+
+![Illumina quality score](../../images/illumina_qs.png "Comparison between different encodings")
+
+Base call quality scores are represented with the Phred range. Different Illumina (formerly Solexa) versions
+used different scores and ASCII offsets. Starting with Illumina format 1.8, the score now represents the standard
+Sanger/Phred format that is also used by other sequencing platforms and the sequencing archives.
+
+![FASTQ quality score](../../images/fastq_qs.png "The ASCII interpretation and ranges of the different Phred score notations used by Illumina and the original Sanger interpretation. Although the Sanger format allows a theoretical score of 93, raw sequencing reads typically do not exceed a Phred score of 60. In fact, most Illumina-based sequencing will result in maximum scores of 41 to 45 (image from <a href='https://en.wikipedia.org/wiki/FASTQ_format'>Wikipedia</a>).")
+
+
+# Assessing quality
+
+This tutorial provides two sample datasets: one generated with Illumina and the other with ElemBio (AVITI) machine. Pick any. We will compare qualities of the two platforms at the and of this tutorial. 
+
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}):
+>
+>    **Element** (AVITI). This datasets contains forward (r1) and reverse (r2) reads for two samples---`s1` and `s2`:
+>
+>    ```
+>    https://zenodo.org/records/10870107/files/elem_s1_r1.fq.gz
+>    https://zenodo.org/records/10870107/files/elem_s1_r2.fq.gz
+>    https://zenodo.org/records/10870107/files/elem_s2_r1.fq.gz
+>    https://zenodo.org/records/10870107/files/elem_s2_r2.fq.gz
+>    ```
+>
+>    **Illumina**. This datasets contains forward (_1) and reverse (_2) reads for two samples---`bl` and `ch`:
+>
+>    ```
+>    https://zenodo.org/record/5119008/files/M117-bl_1.fq.gz
+>    https://zenodo.org/record/5119008/files/M117-bl_2.fq.gz
+>    https://zenodo.org/record/5119008/files/M117-ch_1.fq.gz
+>    https://zenodo.org/record/5119008/files/M117-ch_2.fq.gz
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. Check that the datatype is set to `fastqsanger.gz` (Galaxy usually automatically assigns the right datatype, but check anyway).
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 4. Create a paired collection from your data (relevant part beging at 0:57 and and the case of this tutorial we have 4 datasets): 
+>
+>    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/uZUt9XIHUQo?si=F0qUj76L_lrM7R2j&amp;start=57" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
```suggestion
>    {% snippet faqs/galaxy/collections_build_list_paired.md %}
```

perhaps you'd like to add this video (which is captioned, good!) to that snippet?

Reply:
I really want to keep the video as is at the moment. I will make a new one later. This is a draft anyway